test/e2e/autoscaling/cluster_autoscaler_scalability.go

```diff
@@ -173,10 +178,10 @@ var _ = framework.KubeDescribe("Cluster size autoscaler scalability [Slow]", fun
 		expectedResult := createClusterPredicates(nodeCount + additionalNodes1)
 		config := createScaleUpTestConfig(nodeCount, nodeCount, rcConfig, expectedResult)
 
-		epsilon := 0.05
-
 		// run test #1
-		testCleanup1 := simpleScaleUpTestWithEpsilon(f, config, epsilon)
+		tolerateUnreadyNodes := additionalNodes1 / 20
```

Comment:
why 20? What this constant mean?

Reply:
It used to be 5% (epsilon = 0.05), so I translated this from relative to absolute. I have no clue where the 0.05 number comes from.